.gitignore

```diff
@@ -15,3 +15,5 @@ ci_build/
 .#*
 zig-out/
 .zig-cache/
+target/
+Cargo.lock
```

Comment:
We should commit the lockfiles for reproducibility. Is that right @midnightveil ?